Example/ZcashLightClientSample/ZcashLightClientSample/Send/SendViewController.swift

```diff
@@ -247,6 +247,7 @@ class SendViewController: UIViewController {
                 KRProgressHUD.dismiss()
                 loggerProxy.info("transaction created: \(pendingTransaction)")
             } catch {
+                KRProgressHUD.dismiss()
                 fail(error)
                 loggerProxy.error("SEND FAILED: \(error)")
```

Comment:
```suggestion
                loggerProxy.error("SEND FAILED: \(error)")
                KRProgressHUD.dismiss()
                fail(error)
```
Log failures before doing other things.

Reply:
Updated the code with one more instance a few lines above it.